pkg/notes/service/fetch.ts

```diff
@@ -34,4 +39,17 @@ export class FetchService {
 
     return note;
   }
+
+  async fetchNoteAttachments(
+    noteID: NoteID,
+  ): Promise<Result.Result<Error, Medium[]>> {
+    const attachments =
+      await this.noteAttachmentRepository.findByNoteID(noteID);
+
+    if (Result.isErr(attachments)) {
+      return attachments;
+    }
+
+    return Result.ok(attachments[1]);
```

Comment:
```suggestion
    return attachments;
```
取り出した後に包んでから返しているので, そのまま `return` するのと同じだと思います.